gcp-common/build.gradle

```diff
@@ -3,8 +3,9 @@ plugins {
 }
 
 dependencies {
+    annotationProcessor(mnSerde.micronaut.serde.processor)
     compileOnly(mn.micronaut.discovery.core)
-    implementation(mn.micronaut.jackson.databind)
+    implementation(mnSerde.micronaut.serde.jackson)
```

Comment:
we shouldnt depend on any implementation

Reply:
changed here: https://github.com/micronaut-projects/micronaut-gcp/pull/848/commits/f4fbc94a86a8b93be0ad66f6e70459cc3abb1dc3